frontend/src/metabase/search/containers/SearchApp.styled.tsx

```diff
@@ -6,7 +6,7 @@ import {
 import { Flex, Stack } from "metabase/ui";
 
 const SEARCH_BODY_WIDTH = "90rem";
-const SEARCH_SIDEBAR_WIDTH = "15rem";
+const SEARCH_SIDEBAR_WIDTH = "18rem";
```

Comment:
Expanding sidebar width based on Maz's comments to prevent overflow on the initial state (for English in `Lato` font, anyway)